src/app/shared/models/Label.ts

```diff
@@ -0,0 +1,4 @@
+export class Label {
```

Comment:
How are these classes created and kept up-to-date?
This may be a job for code generation https://github.com/swagger-api/swagger-codegen/tree/master/samples/client/petstore/typescript-angular2